tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -149,4 +156,37 @@ describe("Feature: Sum of grouped answers equal to validation against value sour
       await expect(await $(SecondBreakdownAnswerPage.errorNumber(1)).getText()).toBe("Enter answers that add up to 3");
     });
   });
+  describe("Given I edit a question from a Calculated Summary page", () => {
+    it("When I change the answer and there is a question that needs to be revisited before I can return to the Calculated Summary Page, Then I revisit the relevant page before I route back to the Calculated Summary page", async () => {
+      await $(TotalAnswerPage.total()).setValue("5");
+      await click(TotalAnswerPage.submit());
+
+      await answerBothBreakdownQuestions(["2", "1", "1", "1"], ["1", "2", "0", "0"]);
+
+      await click(AnotherTotalPlaybackPage.submit());
+
+      await expect(browser).toHaveUrlContaining(SubmitPage.pageName);
+
+      await $(SubmitPage.previous()).click();
```

Comment:
ah, I realise I did this in the screen recording, but these two steps aren't required, no need to go to the submit page and then press previous, can just go straight into editing breakdown 1